src/Components/HomePage/HomePage.css

```diff
@@ -1,72 +1,55 @@
-.phone {
-    position: absolute;
-    top: 175px;
-    width: 700px;
-    box-shadow: 0 8px 24px rgba(0,0,0,.2);
-}
-
-
-.background2 {
-    width: 750px;
-    margin-top: 60px;
-    background-color: grey;
-    height: 485px;
-    z-index: -1;
-    position: absolute;
+.homeHero {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    position: relative;
```

Comment:
Flex display allows the easy 50/50 split of the children. (The alternatives are to use float's, or to use a display inline-block with a negative 4 margin, since inline-block elements have an l/r margin of 4px by default.)

Align-items and justify-content ensure the two columns are vertically centered (only works with flex box)